server/registry/names/common.go

```diff
@@ -36,11 +36,6 @@ const (
 // User provided identifiers should be validated according to this format.
 var customIdentifier = regexp.MustCompile(`^[a-z0-9-.]+$`)
 
-// GenerateID generates a random resource ID.
-func GenerateID() string {
```

Comment:
Nothing was using this (I think it was orphaned when we stopped automatically generating IDs), so I lazily decided to delete it rather than test it.